src/SpotifyClient/Playlist/UserPlaylistWrapper.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  ChangePlaylistDetails,
+  UpdatePlaylistItems,
+} from "../../Types/Playlist";
+import IHttpClient from "../IHttpClient";
+
+import PlaylistWrapper from "./PlaylistWrapper";
+
+export default class UserPlaylistWrapper extends PlaylistWrapper {
```

Comment:
Eventually extend all user wrappers with base wrapper. Users authentication lets u access anything, only Application authentication is limited as is.

ref: https://developer.spotify.com/documentation/web-api/concepts/authorization